src/Pmi/Controller/WorkQueueController.php

```diff
@@ -280,6 +281,7 @@ public function exportAction(Application $app, Request $request)
         $params = array_filter($request->query->all());
         $params['_count'] = $pageSize;
         $params['_sort:desc'] = 'consentForStudyEnrollmentAuthored';
+        $params['siteOrganizationId'] = $app->getSiteOrganizationId();
```

Comment:
Can we set this only if the patient status filter is set? Otherwise, we are adding this to every request, and the params array is logged with the export action.

Reply:
And for consistency, maybe make the same change for AJAX requests above, too.